pkg/terraform/stages/azure/stages.go

```diff
@@ -46,3 +55,82 @@ var StackPlatformStages = []terraform.Stage{
 		[]providers.Provider{providers.AzureStack},
 	),
 }
+
+// extractAzureLBConfig extracts the load balancer information from the terraform outputs, generates the
+// Load Balancer Config file, regenerates the bootstrap ignition, and updates the terraform variables file.
+func extractAzureLBConfig(s stages.SplitStage, directory string, terraformDir string, file *asset.File, tfvarsFile *asset.File) (string, error) {
```

Comment:
The AWS version of this function looks like an identical copy. Maybe this could be extracted out from the individual providers and a common function could be used instead?

Reply:
done.